### ISSUE TITLE
<div rend="parallel-left- and -right-__ ">

### ISSUE THREAD

**Issue by wehooper** [IUBLibTech/newton_chymistry#44]
In some documents Newton drew a vertical rule down the center of his page, copying and paraphrasing his source on the left, and writing enumerated notes on the right, with corresponding note markers on the left side.

We wrapped each column in a div with a rend attribute that indicated the dimensions, and, if I recall correctly, I provided a range of options."

See ALCH00008.xml, Keynes 19, "Collectiones ex Novo Lumine Chymico . . ." starting at the top for a good example of the problem and the solution that I worked out with some guidance from John and Tim Bowman.

In ALCH00008.xml, look at lines 416 and 487.

In browsers, P4 : http://webapp1.dlib.indiana.edu/newton/mss/norm/ALCH00008
P5: http://carbon.dlib.indiana.edu:8220/text/ALCH00008/diplomatic#?c=&m=&s=&cv=]

**Comment by Conal-Tuohy** [comment]
done with CSS grid:
```css

.rend-parallel-docs {
	display: grid;
}
.rend-parallel-left {
	grid-column: 1;
}
.rend-parallel-left-30 {
	grid-column: 1;
	width: 30em;
}
.rend-parallel-right {
	grid-column: 2;
}
```

**Comment by Conal-Tuohy** [comment]
addressed in 68eaeba3affbefd406c29b8b8f952ffe23aba113
